### PR TITLE
LPS-175839 buildLang

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/_MenuItem.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'atlas-variables';
 
 .site_navigation_menu_editor_MenuItem {
-	margin-bottom: 1.5rem;
+	margin-bottom: 1rem;
```
Comment (ealonso) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/_MenuItem.scss:4]: @victorg1991 class="mb-1"

Comment (victorg1991) [reply modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/_MenuItem.scss:4]: tru

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
@victorg1991, as we talked, we need a second language key or change it so that it takes into account the plural 😿.

-> 2 Pages "has" been added to the menu

**Comment by victorg1991** [comment]
Done Mr @jkappler

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
Thanks a lot @victorg1991, now we wait for front review ❤️

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by liulukiwi** [comment]
resend with test-fix: https://github.com/liferay-echo/liferay-portal/pull/11206

**Pull request closed without merging**